enclave-runtime/src/lib.rs

```diff
@@ -87,21 +85,18 @@ mod sync;
 mod tls_ra;
 pub mod top_pool_execution;
 
+#[cfg(feature = "teeracle")]
+pub mod teeracle;
+
 #[cfg(feature = "test")]
 pub mod test;
 
-// this is a 'dummy' for production mode
-#[cfg(not(feature = "test"))]
-#[no_mangle]
-pub extern "C" fn test_main_entrance() -> size_t {
-	unreachable!("Tests are not available when compiled in production mode.")
-}
-
```

Comment:
Can you please elaborate on this change?

Reply:
This was just moved out of `lib.rs` to a separate `empty_impls.rs`.

We need these `empty` implementations, because we have functions that are not available with certain compiler flags. But, because of our enclave interface with the EDL file, we always need an implementation, otherwise we have linker errors. So for these cases, we have an empty dummy implementation.

The same now goes for the teeracle, `market_data_update`. That's why I introduced a new module for these empty implementations, to not clutter the `lib.rs` further.

Reply:
Ah thanks for explaining! I did not see that it was moved, I thought it was deleted.